tests/DbTestCase.php

```diff
@@ -455,4 +458,101 @@ protected function disableCapacity(
 
         return $definition;
     }
+
+    /**
+     * Helper method to help creating complex forms using the FormBuilder class.
+     *
+     * @param FormBuilder $builder RuleConfiguration
+     *
+     * @return Form Created form
+     */
+    protected function createForm(FormBuilder $builder): Form
```

Comment:
I guess form specific should not be part of main DbTest case, a FormTest Case should be added

Reply:
I don't known, specific code for the asset genericity tests was put directly into the DBTestCase class too.

I am not against moving the code somewhere else but in this case we should use composition over inheritance.

Reply:
OK, do as you want

Reply:
OK, do as you want

Reply:
Asset definition will have a transverse impact. Indeed, some methods of various itemtypes may have a different behaviour depending on existing assets definitions. I planned to add tests on them later.

You can probably move these methods in a `FormTestTrait`. It can be done later, to not block existing PRs merging.

Reply:
Lets do it later then